### ISSUE TITLE
RCTURLRequestHandlers both reported that they can handle the request

### ISSUE THREAD

**Issue by jbrozena22** [benjreinhart/react-native-aws3#11]
My app is crashing immediately upon uploading a file to S3 with this package. The strange thing is that the request goes through fine, and the file uploads, but the UI on the app crashes with the attached screenshot.

[In digging around for this error](https://github.com/facebook/react-native/issues/4558), it looks like it usually occurs when the URL supplied to the underlying networking request is null. 

Has anyone else here come across this error while using this package?

![simulator screen shot aug 5 2016 11 14 09 am](https://cloud.githubusercontent.com/assets/6846801/17441172/c6674146-5afd-11e6-83a7-3d7b9df6896e.png)

**Comment by benjreinhart** [comment]
I have not seen that. It feels odd to me that this would be because of a `null` url. Looks like we have a (mostly) [hardcoded url](https://github.com/benjreinhart/react-native-aws3/blob/master/src/RNS3.js#L35) [passed to the XMLHttpRequest](https://github.com/benjreinhart/react-native-aws3/blob/master/src/Request.js#L42), not sure how that could be `null`. Also, in the error above, looks like it is showing you a valid library url for the image.

Can you provide your RN version for context?

**Comment by jbrozena22** [comment]
Upgrading RN to v 0.31.0 (latest as of today) solved the problem. This [issue](https://github.com/facebook/react-native/issues/9031) seems to have been resolved by this commit: https://github.com/facebook/react-native/commit/ffb690f072aa7dccb2d21f7b6902f6818abb534d

TLDR - upgrade React Native to 0.31.0

**Comment by jbrozena22** [comment]
And for reference, I was getting this error on React Native v 0.30.0

**Comment by benjreinhart** [comment]
Sweet, thanks @jbrozena22 !
